etc/profile-a-l/audacity.profile

```diff
@@ -33,7 +39,7 @@ noroot
 notv
 nou2f
 novideo
-protocol unix,inet
```

Comment:
Only `inet` indicates audacity may run a local server.

Reply:
See #4659.

Reply:
That flew way under my radar. Fixed, thanks!